test/unit/src/Base64Test.cpp

```diff
@@ -25,8 +20,15 @@ TEST_CASE("Base64")
 		REQUIRE( toBase64( "any carnal pleasur" ) == "YW55IGNhcm5hbCBwbGVhc3Vy" );
 		REQUIRE( toBase64( "any carnal pleasu" ) == "YW55IGNhcm5hbCBwbGVhc3U=" );
 		REQUIRE( toBase64( "any carnal pleas" ) == "YW55IGNhcm5hbCBwbGVhcw==" );
+		REQUIRE( toBase64( "any" ) == "YW55" );
+		REQUIRE( toBase64( "an" ) == "YW4=" );
+		REQUIRE( toBase64( "a" ) == "YQ==" );
+		REQUIRE( toBase64( "" ) == "" );
+		REQUIRE( "" == toString( fromBase64("") ) );
```

Comment:
Style nit: spaces within function braces (see [guide](https://github.com/cinder/Cinder/blob/master/CONTRIBUTING.md#style-guide)). Here and below, and also on the `string ( ...` on line 11.